tensorflow_probability/python/layers/distribution_layer.py

```diff
@@ -1811,7 +1811,7 @@ def __init__(
       mean_fn=None,
       jitter=1e-6,
       convert_to_tensor_fn=tfd.Distribution.sample,
-      name=None):
+      **kwargs):
```

Comment:
We generally avoid using catchall **kwargs this way in TFP. Please instead keep the explicit `name` argument here, and propagate it explicitly to the later call to construct the distribution instance.

The reason we avoid kwargs is because it makes things like docstrings and tab-completion generally less useful.

Reply:
Explicitly passed `name` argument